gloss/gloss.cabal

```diff
@@ -35,11 +35,6 @@ Flag GLFW
   Description:  Enable the GLFW backend
   Default:      False
 
-Flag ExplicitBackend
-  Description:  Expose versions of 'display' and friends that allow
-                you to choose what window manager backend to use.
-  Default:      False
-
```

Comment:
I removed this. No idea why it was still there... I'm pretty sure (like 98% or something) that it's just not doing anything.